bin/edgex-mongo-launch.sh

```diff
@@ -8,22 +8,28 @@
 #Exit the script immediately if a command exits with a non-zero status
 set -e
 
-###
-# Run MongoDB bind to localhost.
-###
-mongod &
-
-###
-# Run Edgex-Mongo Go Application and initiate the database
-###
-cd cmd/
-./edgex-mongo --profile=docker --confdir=res
-
-
-###
-# Restart Edgex-Mongo with enabled authentication and bind_ip_all
-###
-mongod --shutdown
+#hiden file used for determine if the database was already initialized
```

Comment:
fix typo in comment:
```suggestion
#hidden (or sentinel) file used to determine if the database was already initialized
```